tests/phpunit/test-wc-stripe.php

```diff
@@ -109,6 +109,7 @@ function() {
 	}
 
 	private function reloadPaymentGateways() {
+		woocommerce_gateway_stripe()->_TESTS_ONLY__unset_main_stripe_gateway();
```

Comment:
```suggestion
		Closure::bind(
			function () {
				$this->stripe_gateway = null;
			},
			woocommerce_gateway_stripe(),
			WC_Stripe::class
		)();
```
Use this more beautiful hack.
Closure::bind will redefine the scope to WC_Stripe::class and set $this to woocommerce_gateway_stripe() granting access to protected and private properties.

Reply:
Thanks for the suggestion, done in 9e6635b3